### PR TITLE
Reword autoupdate message about having current version to make it l10n friendly

### LINKED ISSUE #5738: Improve version message in Manage Add-ons dialogue
**Describe the bug**
For localization purposes it is not acceptable to paste strings together. Just because one language is comfortable using an order of `a` `:` `b` doesn't mean that another language won't need `b` `:` `c`, nor does it mean that another language might not use a character other than `:`.

**To Reproduce**
1. Go to 'Help>Check for Updates'
2. Read the box for `ZAP Core`

**Expected behavior**
1. en*: Don't have a ` ` before `:`.
2. fr/*: Don't force the use of `:` as the separator
3. en*: It's better to put the message before the version.

**Screenshots**
image

**Software versions**
 - ZAP: 2.8.0
 - OS: Windows 10

**Errors from the zap.log file**

**Additional context**

### PR DESCRIPTION
**Pull request by jsoref**
Fixes #5738

### CODE REVIEW

**Comment by jsoref** [comment]
The bot is apparently not smart enough to detect that `refresh` functions as an equivalent guard.

**Comment by thc202** [comment]
The analysis is correct, since `refresh` is true by default the NPE would happen when `corePanel` is not `null` and `latestInfo` is `null`.

**Comment by kingthorin** [comment]
@jsoref were you going to fix the null dereference?

File: zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties [hunk 1]
```diff
@@ -930,7 +930,7 @@ cfu.check.upd.downloaded   = All updates downloaded, see Output tab for details.
 cfu.check.upd.downloading  = Downloading updates. You can close this dialog and the downloads will carry on in the background.
 cfu.check.zap.downloaded   = ZAP downloaded to {0}
 cfu.check.zap.downloading  = Downloading ZAP. You can close this dialog and the download will carry on in the background.
-cfu.check.zap.latest   = ZAP is on the latest version
+cfu.check.zap.latest   = ZAP is the latest version ({0})
```
Comment (thc202) [inline zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties]: From IRC talk, `ZAP is up-to-date ({0})`.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**